### PR TITLE
GPII-4135: Add new custom button type - key sequencer

### PR DESCRIPTION
**Pull request by Karadaliev**
Added new key sequencer type custom button

* Added new "buttonType": "KEY" in the buttonList's custom type buttons
* Handling the new custom type in gpii.app.generateCustomButton as well, generating "custom-keys" type setting
* Created new gpii.qss.customKeysPresenter for the custom key setting
* Added onQssExecuteKeySequence event which uses Steve's gpii.windows.sendKeys.send() to send the keys to Windows

### CODE REVIEW

**Comment by cindyli** [comment]
ok to test

**Comment by Karadaliev** [comment]
@amb26 Please let me know when https://github.com/GPII/windows/pull/282 is merged with the master (and we have the new package version) in order to update this PR with the proper package 🛩 .

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
         "electron": "3.0.2",
         "electron-edge-js": "8.3.8",
         "electron-localshortcut": "3.1.0",
-        "gpii-windows": "0.3.0-dev.20191017T162536Z.acb3d40",
+        "gpii-windows": "stegru/windows#GPII-4135",
```
Comment (amb26) [inline package.json]: Cheers - can you please update this to the latest windows release of 0.3.0-dev.20191106T150515Z.cadcb7c which should incorporate the changes from @stegru

Comment (Karadaliev) [reply package.json]: Done 💯

**Pull request merged**